xstate-chatbot/nodejs/src/machine/hospital-details.js

```diff
@@ -0,0 +1,1068 @@
+            },
+            {
+              cond: (context) => context.validMessage == true,
+              target: '#l2l3Hospital',
+            },
+            {
+              target: 'error',
+            },
+          ],
+        },
+        error: {
+          onEntry: assign((context, event) => {
+            dialog.sendMessage(context, dialog.get_message(messages.noUserFetch, context.user.locale), false);
+          }),
+          always: 'prompt',
+        },
+      },
+    },//Hospital Type Id 1/2/3 
+    l2Hospital: {
+      id: 'l2Hospital',
+      initial: 'process',
+      states: {
+        prompt: {
+          onEntry: assign((context, event) => {
+
+          }),
+          on: {
+            USER_MESSAGE: 'process',
+          },
+        },
+        process: {
+          onEntry: assign((context, event) => {
+            if (new Date().getHours() >= 14 && new Date().getHours() <= 18) {
+              context.isValid = true;
+            } else {
+              context.isValid = false;
+            }
+          }),
+          always: [
+            {
+              cond: (context) => context.isValid == false,
+              target: '#l2AvailableOxygenBeds',
+            },
+            {
+              cond: (context) => context.isValid == true,
+              target: '#l2HospitalUpdate',
+            },
+          ],
+        },
+        error: {
+          onEntry: assign((context, event) => {
+            dialog.sendMessage(context, dialog.get_message(messages.l2HospitalUpdate.error, context.user.locale), false);
+          }),
+          always: 'prompt',
+        },
+      },
+    },// L2 Hospital comparing time slots
+    l2HospitalUpdate: {
+      id: 'l2HospitalUpdate',
+      initial: 'prompt',
+      states: {
+        prompt: {
+          onEntry: assign((context, event) => {
+            context.grammer = grammers.binaryChoice.grammer;
+            let message = dialog.get_message(messages.l2HospitalUpdate.prompt, context.user.locale);
+            message += dialog.get_message(grammers.binaryChoice.prompt, context.user.locale);
+            dialog.sendMessage(context, message);
+          }),
+          on: {
+            USER_MESSAGE: 'process',
+          },
+        },
+        process: {
+          onEntry: assign((context, event) => {
+            context.intention = dialog.get_intention(context.grammer, event, true);
+          }),
+          invoke: {
+            src: (context, event) => bedsService.getHospitalById(context.slots.hospital.mobileNumber),
+            onDone: [
+              {
+                cond: (context, event) => context.intention === 'YES',
+                actions: assign((context, event) => {
+                }),
+                target: '#l2ConfirmedCOVIDPatientsOnOxygen',
+              },
+              {
+                cond: (context, event) => context.intention === 'NO',
+                actions: assign((context, event) => {
+                }),
```

Comment:
Same as the previous comment and make changes at all other places

Reply:
removed